asm/pokemon_3.s

```diff
@@ -307,7 +307,6 @@ _0803FB56:
 	bx r0
 	thumb_func_end DrawSpindaSpots
 
-	.if REVISION >= 1
```

Comment:
@YamaArashi Except for the `GAME_LANGUAGE` candidate, is this function actually any different?